pkg/aggregator/aggregator.go

```diff
@@ -336,6 +339,9 @@ func NewBufferedAggregator(s serializer.MetricSerializer, eventPlatformForwarder
 		tlmContainerTagsEnabled:     pkgconfigsetup.Datadog().GetBool("basic_telemetry_add_container_tags"),
 		agentTags:                   tagger.AgentTags,
 		globalTags:                  tagger.GlobalTags,
+		enrichTags:                  tagger.EnrichTags,
```

Comment:
Is there a reason to store the method in a field? `agentTags` and `globalTags` are overridden in tests, but not this one. I think it is a legacy pre-components pattern, no need to copy it if we're not using it.

Reply:
Sounds good. I wasn't sure what pattern to follow. I will update the code